### DISCUSSION TITLE
Welcome to mailing Discussions!

### DISCUSSION THREAD

**Discussion by psugihara** [sofn-xyz/mailing#104] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by mrlubos** [comment]
Hey! 👋

I stumbled upon mailing by chance and immediately recognised its potential. I've had a similar feeling about Recoil when they released the first open source version. If mailing is anything like Recoil, it has a bright future.

I actually created my custom version of mailing before this project existed. It uses a similar stack (mjml + react-mjml) so I'm familiar with the current email development workflow pain points. I didn't dedicate enough resources to my implementation to make it open source, so I'm delighted to see more people working on this problem. mjml-react isn't very active whereas I am seeing good engagement in this project, I think that's important.

I'd love for mailing to become a de facto standard for email development. It's good that you started with mjml in React as it has solid tools, but there's no reason it couldn't eventually work with Vue, Angular, and other popular frameworks. If possible, I would be interested in contributing to this project, it has great potential.

As for my introduction, I am going to plug [my website](https://lmen.us/) since it contains that and more. I write software (duh) with particular focus on user experience, outside software I am into sports, finance, and various weird interests. Glad to be here and can't wait to see where this project goes!
